interfaces/Types.ts

```diff
@@ -23,6 +23,7 @@ export enum PipelineType {
 	"audio-source-separation" = "audio-source-separation",
 	"voice-activity-detection" = "voice-activity-detection",
 	/// computer vision
+	"image-captioning" = "image-captioning",
```

Comment:
`image-to-text` if we expect other more general models that do this? What do you think?

Reply:
Totally. Will update the definition to `image-to-text`